structs.go

```diff
@@ -74,6 +74,15 @@ func New(s interface{}) *Struct {
 // fields will be neglected.
 func (s *Struct) Map() map[string]interface{} {
 	out := make(map[string]interface{})
+	s.FillMap(out)
+	return out
+}
+
+// FillMap like Map, but not return map, should pass map
+func (s *Struct) FillMap(out map[string]interface{}) {
```

Comment:
Description should be `FillMap is the same as Map. Instead of returning the output, it fills the given input`